views.go

```diff
@@ -365,6 +365,15 @@ func (lk *logkeeper) viewBuild(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.FormValue("metadata") == "true" {
```

Comment:
if we have this do we still need the `metadata` option on the `build/{build_id}/all` route?

Reply:
I was thinking that. Honestly I think it makes more sense for the `all` route to have the list of build + test metadata, but to make it consistent with this route I just put it here (since this is the "view build" route). If I could create it from scratch I would have just two routes: `build/{build_id}` and `build/{build_id}/test/{test_id}` which would show either all logs or the test logs, respectively, and both would accept a metadata query param.

Reply:
I do think @khelif96 is using the `metadata` in the `all` route so we probably can't remove it.